timesync/clock.go

```diff
@@ -117,6 +127,15 @@ func (t *NodeClock) tick() {
 
 	layer := t.TimeToLayer(t.clock.Now())
 
+	d := layer.Difference(t.lastTickedLayer)
```

Comment:
do you know why lastTickedLayer is updated here?
https://github.com/spacemeshos/go-spacemesh/blob/d59cadce8a6c80721b2cbebd0a7f5872ec9b65e2/timesync/clock.go#L176
@fasmat

Reply:
But now that you mention it, the check is incorrect. It should update `lastTickedLayer` if the layer awaited is before it not after. I fixed it in this PR:

https://github.com/spacemeshos/go-spacemesh/pull/4047